mongodbforms/documents.py

```diff
@@ -308,6 +308,10 @@ def __new__(cls, name, bases, attrs):
         if 'media' not in attrs:
             new_class.media = media_property(new_class)
 
+        # Django 1.6 support
+        if not hasattr(new_class, 'declared_fields'):
+            new_class.declared_fields = attrs['base_fields']
```

Comment:
Another option would be calling Django's get_declared_fields directly, but that will result in incompatibility with Django >= 1.9.

Surely we can do import inside of the condition, but I considered it to be a bit worse than the current solution.